### PR TITLE
Post 1.5.2 release

### PR DESCRIPTION
**Pull request by schmichael**

### CODE REVIEW

**Review by tgross — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -175,6 +202,19 @@ BUG FIXES:
...
+IMPROVEMENTS:
+
+* build: Update to go1.20.2 [[GH-16427](https://github.com/hashicorp/nomad/issues/16427)]
+*
+* BUG FIXES:
+*
+* * client: Fixed a bug where clients using Consul discovery to join the cluster would get permission denied errors [[GH-16490](https://github.com/hashicorp/nomad/issues/16490)]
+* * client: Fixed a bug where cpuset initialization fails after Client restart [[GH-16467](https://github.com/hashicorp/nomad/issues/16467)]
+* * plugin: Add missing fields to `TaskConfig` so they can be accessed by external task drivers [[GH-16434](https://github.com/hashicorp/nomad/issues/16434)]
+* * services: Fixed a bug where a service would be deregistered twice [[GH-16289](https://github.com/hashicorp/nomad/issues/16289)]
```
Comment (lgfa29) [inline CHANGELOG.md:216]: I think your editor may have added more `*` than needed. Here's the rendered version:
https://github.com/hashicorp/nomad/blob/7177b9a7db3f4226ab9f7b88773c35374b03b0a9/CHANGELOG.md#147-march-21-2023

**Review by lgfa29 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (schmichael) [reply CHANGELOG.md]: Good catch! Sorry about that.

**Review by lgfa29 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (lgfa29) [reply CHANGELOG.md]: No worries, I'm always annoyed that this is still not an automated process 😅

**Pull request merged**